content/general/concepts/cicd-pipeline/cicd-pipeline.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: 'CICD Pipeline'
+Description: 'CICD Pipeline automates the software / application delivery process.'
+
+Subjects: 
```

Comment:
These are not valid subjects: subjects come from [this](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md) list.

Reply:
Thank you, I will look these up for the appropriate subjects.